samples/client/echo_api/php-nextgen/.openapi-generator/FILES

```diff
@@ -24,13 +24,13 @@ docs/Model/TestQueryStyleDeepObjectExplodeTrueObjectAllOfQueryObjectParameter.md
 docs/Model/TestQueryStyleFormExplodeTrueArrayStringQueryObjectParameter.md
 git_push.sh
 phpunit.xml.dist
+src/ApiException.php
 src/Api/AuthApi.php
 src/Api/BodyApi.php
 src/Api/FormApi.php
 src/Api/HeaderApi.php
 src/Api/PathApi.php
 src/Api/QueryApi.php
-src/ApiException.php
```

Comment:
are you using git bash or WSL to run the sample update script under windows?

Reply:
I am using git bash

Reply:
can you try WSL instead?